src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -1,11 +1,68 @@
+# rubocop:disable Metrics/ModuleLength
 module Webui::NotificationHelper
+  TRUNCATION_LENGTH = 100
+  TRUNCATION_ELLIPSIS_LENGTH = 3 # `...` is the default ellipsis for String#truncate
+
   def link_to_show_less_or_more
     parameters = params.slice(:show_more, :state, :project).permit!
     less_or_more = parameters[:show_more] ? 'less' : 'more'
     parameters[:show_more] = parameters[:show_more] ? nil : '1'
     link_to("Show #{less_or_more}", my_notifications_path(parameters))
   end
 
+  # TODO: Content of ViewComponent. Move to sub-classes once STI is set.
+  def excerpt(notification)
```

Comment:
Shouldn't we use the newly introduced `NotificationExcerptHelper` for this?

Reply:
Oh I see you are removing it...

Reply:
Yes! I introduced another commit removing the NotificationExcerptHelper. I'm finally not going to add a helper per method, but everything goes to NotificationHelper.